ui/src/app/pages/analysis/runtime-panel.tsx

```diff
@@ -1255,7 +1255,19 @@ const DataProcConfigSelector = ({
         </FlexRow>
         <FlexRow style={styles.labelAndInput}>
           <label style={styles.label} htmlFor='num-preemptible'>
-            Preemptible
+            Preemptible workers
+            <TooltipTrigger
+              content={
+                <div>
```

Comment:
We can use just text here as value of content rather than <div>
eg content = {`Preemptible secondary workers can be added in addition to
                  prim....`}

Reply:
Done 👍